cypress/support/commands.js

```diff
@@ -1,38 +1,11 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-
- 
-
-Cypress.Commands.add('login',(username, password)=>{
-    cy.visit('https://www.saucedemo.com/')
-    cy.get('[data-test="username"]').type(username)
-    cy.get('[data-test="password"]').type(password)
-    cy.get('[data-test="login-button"]').click()
+ // import LandingPage from './page-objects/Landing.page.js'
+// Cypress.Commands.add('login',(username, password)=>{
```

Comment:
why is this commented out :) I am not a cypress expert, so I really don't know what is  better and whether it influences the method you have implemented in the Landing page class